workspace/__lib__/xod-dev/esp8266-mcu/send/patch.cpp

```diff
@@ -0,0 +1,27 @@
+
+struct State {
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    if (!isInputDirty<input_SND>(ctx))
+        return;
+
+    auto client = getValue<input_CLNT>(ctx);
+    auto msg = getValue<input_MSG>(ctx);
+
+    size_t lastWriteSize;
+
+    for (auto it = msg->iterate(); it; ++it) {
```

Comment:
Produces warning:
`warning: 'xod::List<T>* xod::List<T>::operator->() [with T = char]' is deprecated`
So just replace with `msg.iterate()`

Reply:
Thanks! Fixed.